### PR TITLE
dmd: install necessary shared library to target directory

### PR DESCRIPTION
**Pull request by Char-Aznable**

### CODE REVIEW

File: Formula/dmd.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@ def install
 
     (include/"dlang/dmd").install Dir["druntime/import/*"]
     cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
-    lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
+    lib.install Dir["druntime/**/libdruntime.*", "phobos/**/libphobos2.a", "phobos/**/libphobos2.so*"]
```
Comment (iMichka) [inline Formula/dmd.rb]: Interesting, this does not install the same things than on mac. I would prefer to have this in a separate line if possible; leaving the initial install line pristine:
`lib.install Dir["druntime/**/libdruntime.*", "phobos/**/libphobos2.so*"] unless OS.mac?`

**Comment by iMichka** [comment]
@Char-Aznable Sorry, had forgotten this. Had a lot of other stuff going on lately. Can you rebase this, and maybe squash your 3 commits? Thanks.

**Comment by Char-Aznable** [comment]
@iMichka Done.

**Comment by iMichka** [comment]
Err now you have 6 commits and some merge commits. I was expecting this to be only 1 commit.
I think you can fix this with an interactive rebase. I can do it for you if you do not feel confident to do it. Or you can also open a new pull request.

**Comment by Char-Aznable** [comment]
@iMichka Oops... Now it should be fine.

File: Formula/dmd.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class Dmd < Formula
   desc "D programming language compiler for macOS"
   homepage "https://dlang.org/"
+  revision 2 unless OS.mac?
```
Comment (iMichka) [inline Formula/dmd.rb]: This should be `revision 1 unless OS.mac?`

**Comment by iMichka** [comment]
```
== /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/dmd.rb ==
C: 76:  5: Use 2 (not 3) spaces for indentation.
C: 78:  5: Use 2 (not 3) spaces for indentation.

1 file inspected, 2 offenses detected
==> FAILED
```

**Comment by iMichka** [comment]
@Char-Aznable Thanks for your contribution (and for your patience)!

**Pull request closed without merging**